src/modules/store.js

```diff
@@ -0,0 +1,51 @@
+export default class Store {
+  static getList() {
+    let list = [];
+    if (localStorage.getItem('task') === null) {
+      list = [];
+    } else {
+      list = JSON.parse(localStorage.getItem('task'));
+    }
+    return list;
+  }
+
+  static add(task) {
+    const list = Store.getList();
+    list.push(task);
+    localStorage.setItem('task', JSON.stringify(list));
+  }
+
+  static remove(index) {
+    const list = Store.getList();
+    list.forEach((task) => {
+      if (task.index === index) {
+        list.splice(index - 1, 1);
+        // list = list.filter((task) => task.index !== index);
```

Comment:
Please ensure that all commented codes are removed from your codebase. It makes your codebase clean.